Sources/ApolloWebSocket/DefaultImplementation/WebSocket.swift

```diff
@@ -813,7 +813,8 @@ public final class WebSocket: NSObject, WebSocketClient, StreamDelegate, WebSock
 
       if receivedOpcode == .connectionClose {
         var closeReason = "connection closed by server"
-        if let customCloseReason = String(data: data, encoding: .utf8) {
+        if let customCloseReason = String(data: data, encoding: .utf8),
+           customCloseReason.isEmpty {
```

Comment:
I think this is inverted - this would mean that if the custom close reason is empty, it would fall into the `if`, while if it was not empty, it'd fall into the default case. 

There's a `.apollo.isNotEmpty` extension floating around somewhere in `ApolloUtils` that would make this clearer than just a `!`, fwiw.

Reply:
🤦‍♂️ 

This is what I get for trying to write one line of code without a test. I'll fix it up.

Reply:
Thanks for this. Could we get a simple test added for it as well? I'd be happy to merge this once we have a test!